packages/react-ui/src/localization/id.js

```diff
@@ -33,14 +33,33 @@ const locales = {
         clear: 'Bersihkan'
       },
       legend: {
-        by: 'Dengan {attr}',
-        layerOptions: 'Opsi lapisan',
-        hide: 'Sembunyikan',
-        show: 'Tampilkan',
-        layer: 'lapisan',
-        opacity: 'Opasitas',
-        hideLayer: 'Sembunyikan lapisan',
-        showLayer: 'Tampilkan lapisan'
+        by: 'By {attr}',
+        layerOptions: 'Layer options',
+        hide: 'Hide',
+        show: 'Show',
+        layer: 'layer',
+        opacity: 'Opacity',
+        hideLayer: 'Hide layer',
+        showLayer: 'Show layer',
+        open: 'Open legend',
+        close: 'Close',
+        collapse: 'Collapse layer',
+        expand: 'Expand layer',
+        zoomLevel: 'Zoom level',
+        zoomLevelTooltip: 'This layer is only visible at certain zoom levels',
+        lowerThan: 'lower than',
+        greaterThan: 'greater than',
+        and: 'and',
+        zoomNote: 'Note: this layer will display at zoom levels',
+        notSupported: 'is not a known legend type',
+        subtitles: {
+          proportion: 'Radius range by',
+          icon: 'Icon based on',
+          strokeColor: 'Stroke color based on',
+          color: 'Color based on'
+        },
+        max: 'Max',
+        min: 'Min'
```

Comment:
We need to change English to Indonesian

Reply:
All of it? I for sure don't know Indonesian

Reply:
ChatGPT? haha

Reply:
@gandeszahara can help here

Reply:
hi! I can help for the translation. will check and discuss further with Juan